python/mdvtools/conversions.py

```
@@ -424,16 +538,22 @@ def create_regulamentary_project(
     )
     return p
 
-
-def _create_dt_heatmap(folder, project, mdv, marks):
+def _create_dt_heatmap(
+        project,
+        matrix,
+        ds="elements",
+        ifields = None
+):
+    if ifields == None:
```

Ruff (0.11.9)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)